src/Comet/Styles/Material/MaterialStyle.cs

```diff
@@ -86,7 +86,6 @@ public MaterialStyle(ColorPalette colorPalette)
 			ProgressBar = new ProgressBarStyle
 			{
 				ProgressColor = colorPalette.P500,
-				TrackColor = colorPalette.P100,
```

Comment:
Why is this being deleted?

Reply:
MAUI API changed, there's no TrackColor property in Progress class